lib/ansible/modules/network/ios/ios_acl.py

```diff
@@ -0,0 +1,1430 @@
+                      v2host_report:
+                        description: IGMPv2 Membership Report(5)
+                        type: bool
+                      v2leave_group:
+                        description: IGMPv2 Leave Group(6)
+                        type: bool
+                      v3host_report:
+                        description: IGMPv3 Membership Report(9)
+                        type: bool
+                  ip:
+                    description: Any Internet Protocol.
+                    type: bool
+                  ipv6:
+                    description: Any IPv6.
+                    type: bool
+                  ipinip:
+                    description: IP in IP tunneling.
+                    type: bool
+                  nos:
+                    description: KA9Q NOS compatible IP over IP tunneling.
+                    type: bool
+                  ospf:
+                    description: OSPF routing protocol.
+                    type: bool
+                  pcp:
+                    description: Payload Compression Protocol.
+                    type: bool
+                  pim:
+                    description: Protocol Independent Multicast.
+                    type: bool
+                  sctp:
+                    description: Stream Control Transmission Protocol.
+                    type: bool
+                  udp:
+                    description: User Datagram Protocol.
+                    type: bool
+                  tcp:
+                    description: Match TCP packet flags
+                    type: dict
+                    suboptions:
+                      set:
+                        description:
+                          - When TCP flags are not specified and user want to set
+                            TCP only as protocol option.
+                          - Note, not required if flags options are set.
+                        type: bool
+                      ack:
+                        description: Match on the ACK bit
+                        type: bool
+                      established:
+                        description: Match established connections
+                        type: bool
+                      fin:
+                        description: Match on the FIN bit
+                        type: bool
+                      psh:
+                        description: Match on the PSH bit
+                        type: bool
+                      rst:
+                        description: Match on the RST bit
+                        type: bool
+                      syn:
+                        description: Match on the SYN bit
+                        type: bool
+                      urg:
+                        description: Match on the URG bit
+                        type: bool
+              source:
+                description: Specify the packet source.
+                type: dict
+                suboptions:
+                  address:
```

Comment:
Can we specify a `prefix`, i.e., '192.168.1.0/24', as source/destination?

Reply:
The `prefix` key, in that case, would be mutually exclusive with all other keys under source.

Reply:
yes, It can be added and yes it gives user 1 more option of adding it and I'll add the respective implemenntation. But IMO all such things should be added in respective resource models, and we should strictly stick to the same and if there's any idea that we think post freezing model make sense, we should discuss and bring this to team's notice.

Reply:
Tried it from IOS standpoint but It cannot be added for ios as it doesn't support/expect of adding source/destination in form of address/prefix, for ref:
```
an-ios-01(config-ext-nacl)#10 deny icmp ?
  A.B.C.D       Source address
  any           Any source host
  host          A single source host
  object-group  Source network object group

an-ios-01(config-ext-nacl)#10 deny icmp any ?
  A.B.C.D       Destination address
  any           Any destination host
  host          A single destination host
  object-group  Destination network object group
```